Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -4922,6 +4926,11 @@ private void InitSchedulerRuntimeState(LoggingContext loggingContext, SchedulerS
             {
                 Contract.Requires(loggingContext != null);
 
+                // Start loading data for pip two phase cache and running time table. No need to wait since any operation against the components
+                // will block until the required component is ready.
+                m_pipTwoPhaseCache.StartLoading(waitForCompletion: false);
+                m_runningTimeTableTask.Start();
+

```

Comment:
In the past, these two were getting loaded earlier. I did not understand the reason why we postpone them.